src/Support/Contracts/FieldBuilderDirective.php

```diff
@@ -2,14 +2,18 @@
 
 namespace Nuwave\Lighthouse\Support\Contracts;
 
+use GraphQL\Type\Definition\ResolveInfo;
+
 interface FieldBuilderDirective extends Directive
 {
     /**
      * Add additional constraints to the builder.
      *
      * @param  \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder  $builder  the builder used to resolve the field
+     * @param  array<string, mixed>  $args
+     * @param  \Nuwave\Lighthouse\Support\Contracts\GraphQLContext  $context
```

Comment:
redundant, only add PHPDocs when they add information

Reply:
Not as redundant as they seem, they fix phpstan errors would you mind giving me some short descriptions on `$args` and `$context` so I can add it and make it more useful for users too?

Reply:
It is only necessary for `$args`. Docs are at https://lighthouse-php.com/master/api-reference/resolvers.html#resolver-function-signature

Reply:
@spawnia Thanks for this really helped with the description!